Kconfig.zephyr

```
@@ -292,6 +292,17 @@ config LINKER_LAST_SECTION_ID_PATTERN
 	  be used.
 	  The size of the pattern must not exceed 4 bytes.
 
+config LINKER_USE_RELAX
+	bool "Use --relax Linker option"
```

Please remove `use` as per guide lines:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

On top of that, the `--relax` is a bit ld centric, maybe a title like:
```suggestion
	bool "Linker optimization of call addressing"
```